src/NovelRT/Windowing/WindowingService.cpp

```diff
@@ -113,6 +115,20 @@ namespace NovelRT::Windowing {
 
 
   }
+
+  void WindowingService::checkForOptimus() {
```

Comment:
Is there an equivalent on Linux? What about systems that have both AMD and Nvidia?

Reply:
Linux equivalent is apparently related to the NVIDIA FOSS driver, which hasn't been figured out yet.